### PR TITLE
Fix colors for default light mode theme

### PR DESCRIPTION
**Pull request by hawkeye217**
- Add new color `background_alt`
- Use `primary` instead of `primary-foreground` so any new shadcn components are happy without changes
- Change secondary button variant to be default button
- Change filters button to `selected` when filtering
- Use `selected` on drawer buttons on mobile when filtering or changing calendar dates

### CODE REVIEW

**Review by NickM-27 — APPROVED** [review]

File: web/src/components/filter/ReviewFilterGroup.tsx [hunk 1]
```diff
@@ -682,12 +685,11 @@ function ShowMotionOnlyButton({
       <div className="block md:hidden">
         <Button
           size="sm"
-          variant="secondary"
           className={`duration-0 ${motionOnlyButton ? "bg-selected hover:bg-selected" : "bg-secondary hover:bg-secondary/80"}`}
           onClick={() => setMotionOnlyButton(!motionOnlyButton)}
         >
           <FaRunning
-            className={`${motionOnlyButton ? "fill-primary dark:fill-primary-foreground" : "text-muted-foreground"}`}
+            className={`${motionOnlyButton ? "text-background dark:text-primary" : "text-secondary-foreground"}`}
```
Comment (NickM-27) [inline web/src/components/filter/ReviewFilterGroup.tsx:692]:
```suggestion
            className={`${motionOnlyButton ? "text-white" : "text-secondary-foreground"}`}
```

Comment (hawkeye217) [reply web/src/components/filter/ReviewFilterGroup.tsx:692]: Didn't want to use `text-white` so that any themes can have unique colors later.

Comment (NickM-27) [reply web/src/components/filter/ReviewFilterGroup.tsx:692]: but right now we are putting this on top of selected background. IMO any theme that changes the selected color white will probably look good on it. If not then we should create selected-foreground and make that white

Comment (NickM-27) [reply web/src/components/filter/ReviewFilterGroup.tsx:692]: I added a PR onto this branch for that change https://github.com/blakeblackshear/frigate/pull/10914

**Pull request merged**